requirements.txt

```diff
@@ -1,15 +1,5 @@
 black==24.4.2
 isort==5.13.2
-pytest==8.2.1
 pre-commit==3.7.1
-playwright==1.44.0
```

Comment:
We'll need playwright installed to run the test suite locally, right..?

Reply:
Or is this installed via npm...?

Reply:
it's installed via `npx playwright install chromium` which is already part of the build process in CI.

I don't mind keeping that in if that's how you get it but it's redundant (see our Makefile)

Reply:
https://github.com/pyscript/pyscript/pull/2187/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R58